app/src/main/kotlin/net/primal/android/nostr/notary/NostrNotary.kt

```
@@ -35,7 +34,12 @@ class NostrNotary @Inject constructor(
     private fun findNsecOrThrow(pubkey: String): String {
         return try {
             val npub = Hex.decode(pubkey).toNpub()
-            credentialsStore.findOrThrow(npub = npub).nsec
+            val nsec = credentialsStore.findOrThrow(npub = npub).nsec
+            if (nsec == null) {
```

Can you do `?: throw MissingPrivateKeyException()` here?